src/protocols/rdp/settings.c

```diff
@@ -1529,7 +1549,20 @@ void guac_rdp_push_settings(guac_client* client,
         case GUAC_SECURITY_ANY:
             rdp_settings->RdpSecurity = TRUE;
             rdp_settings->TlsSecurity = TRUE;
-            rdp_settings->NlaSecurity = guac_settings->username && guac_settings->password;
+
+            /* Explicitly disable NLA if FIPS mode is enabled - it won't work */
+            if (guac_fips_enabled()) {
+
+                guac_client_log(client, GUAC_LOG_WARNING,
+                        "Disabling NLA security mode when FIPS mode is enabled.");
```

Comment:
1. This should be at the info level. Nothing is failing or anticipated to fail here (yet); it's just a noteworthy piece of information.
2. As written, this is always true. I suggest rephrasing to explicitly note the recognition of system state and the action taken: `FIPS mode is enabled. Excluding NLA security mode from security negotiation (see: https://github.com/FreeRDP/FreeRDP/issues/3412).`

Reply:
Ah oops, I had updated this to be INFO, but that change got lost somewhere. I'll put it back.